### PR TITLE
WSTEAMA-1179 - Update Billboard component to meet UX designs

### PR DESCRIPTION
**Pull request by holchris**
Resolves JIRA 1179 - https://jira.dev.bbc.co.uk/browse/WSTEAMA-1179

Overall changes
======
Adapt Billboard component to match screenreader UX 

Code changes
======
- LiveLabel and heading onto separate lines 
- Edit padding to ensure component meets UX designs
- Update unit tests

### CODE REVIEW

File: src/app/components/Billboard/index.styles.ts [hunk 1]
```diff
@@ -5,10 +5,18 @@ export default {
...
       '&:hover, &:focus': {
-        color: palette.WHITE,
         textDecoration: 'underline',
       },
+      '::before': {
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
```
Comment (karinathomasbbc) [inline src/app/components/Billboard/index.styles.ts:18]: Does it still work if we remove these? (If not, leave them, I was more curious than anything!)

File: src/app/components/Billboard/index.test.tsx [hunk 2]
```diff
@@ -41,14 +41,14 @@ describe('Billboard', () => {
         altText={imageAlt}
       />,
     );
-    const heading = screen.getByText(title);
-    const messageBannerEl = getByRole('region');
-    expect(messageBannerEl.getAttribute('aria-labelledby')).toBe(
+    const heading = screen.getByRole('heading', { level: 2, name: title });
```
Comment (karinathomasbbc) [inline src/app/components/Billboard/index.test.tsx:44]: Oooh I like this!

**Review by karinathomasbbc — APPROVED** [review]
Excellent work @holchris and @louisearchibald - thank you for this!

File: src/app/components/Billboard/index.styles.ts (hunk 1 above)
Comment (louisearchibald) [reply src/app/components/Billboard/index.styles.ts:18]: Clickable area stops working without them annoyingly 🤔

Comment (karinathomasbbc) [reply src/app/components/Billboard/index.styles.ts:18]: Cool, I just wanted to check, thanks for clarifying!

**Review by amoore108 — COMMENTED** [review]
Looks good. Does this need a11y checked for the re-ordering of the heading and anchor tags, as well as the `z-index` adjustments?

**Review by amoore108 — APPROVED** [review]

**Comment by greenc05** [comment]
Are the image proportions correct without CSS? Screenshot 2024-04-29 at 14 33 06

**Comment by greenc05** [comment]
The black space at the bottom changes/increase as you zoom, this screen shot is at 133% zoom? Screenshot 2024-04-29 at 14 34 31

**Comment by greenc05** [comment]
We have the transparent border (see red line in screen shot) from the live header, there is some acceptance criteria for this which isn't met Screenshot 2024-04-29 at 14 37 01

**Comment by greenc05** [comment]
On larger breakpoints when the black colour bleeds out to the sides, the clickable area is confined to the area in the middle of the 2 yellow borders in the screenshot here, are UX ok witih this? Screenshot 2024-04-29 at 14 40 57

**Comment by greenc05** [comment]
Couple of comments for you, though great work getting the larger clickable area @holchris  @louisearchibald

**Comment by holchris** [comment]
> The black space at the bottom changes/increase as you zoom, this screen shot is at 133% zoom? Screenshot 2024-04-29 at 14 34 31

The black space is a known error that is part of the LiveHeader work, this will be addressed later on! Conversation around this is in this slack thread [here](https://bbc.enterprise.slack.com/archives/C03S8UQJZ6U/p1714389200392989?thread_ts=1714388516.300079&cid=C03S8UQJZ6U)

**Comment by karinathomasbbc** [comment]
I did notice that adding padding to the component has changed the snapshot - see https://bbc.chromatic.com/test?appId=5d28eb3fe163f6002046d6fa&id=662faa72f435502c57b207bc - I wonder if we add the padding to the Curation component instead of the Billboard? Or do we address this in the next PR?

**Comment by holchris** [comment]
> I did notice that adding padding to the component has changed the snapshot - see https://bbc.chromatic.com/test?appId=5d28eb3fe163f6002046d6fa&id=662faa72f435502c57b207bc - I wonder if we add the padding to the Curation component instead of the Billboard? Or do we address this in the next PR?

@karinathomasbbc we can address this in the next PR. @louisearchibald and I have been trying out adding the padding to the curation, but this is a little more tricky than anticipated.

**Pull request merged**